monix-catnap/shared/src/test/scala/monix/catnap/CancelableFSuite.scala

```diff
@@ -17,14 +17,14 @@
 
 package monix.catnap
 
-import cats.effect.IO
+import cats.effect.{SyncIO}
 import minitest.SimpleTestSuite
 
 object CancelableFSuite extends SimpleTestSuite {
   test("apply") {
     var effect = 0
-    val task = IO { effect += 1 }
-    val ref = CancelableF[IO](task)
+    val task = SyncIO {effect += 1 }
+    val ref = CancelableF[SyncIO](task)
 
     val cf = ref.unsafeRunSync()
```

Comment:
I've changed all cancelable tests to use SyncIO b/c unsafeRunSync simply doesn't exist on IO when built for JS.